cypress/support/index.js

```diff
@@ -36,6 +36,18 @@ beforeEach(() => {
 
             cy.get('.btn').click()
         }
+        cy.wait(2000)
```

Comment:
wait statement going in here. the conditional testing below needs to get accurate hit on the elements in body. without the wait, the condition would be checked instantly after signup/login which wouldn't give enough time for onboarding wizard to appear